src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1109,6 +1112,9 @@
         - name: --node-provisioning-mode
           type: string
           short-summary: Set the node provisioning mode of the cluster. Valid values are "Auto" and "Manual". For more information on "Auto" mode see aka.ms/aks/nap.
+        - name: --ssh-access
+          type: string
+          short-summary: Update SSH setting for all node pools in this cluster. Allowed values are ["disabled", "localuser"]. Use "disabled" to disable SSH access, "localuser" to enable SSH access using private key. Default value is "localuser".
```

Comment:
Judging from the code in `_params.py`, in the update scenario, there is no default value? same for nodepool update

Reply:
yep, in update command, we should not set default, otherwise it will always overwrite existing setting.

Reply:
1. If this param is enumerable, please define it with `arg_type=get_enum_type(["disabled", "localuser"])` in `_params.py`. After defining with it, it is not necessary to specify `Allowed values are ["disabled", "localuser"]. ` in the help message manually, and meanwhile, it would also verify users' input automatically.
2. Since this param has been defined with `default=CONST_SSH_ACCESS_LOCALUSER` in `_params.py`, it is not necessary to specify `Default value is "localuser".` manually and it would be added automatically.

Reply:
One more thing, in updating scenario, it can be an empty string without any default value, in this case, we should keep its current value unchanged. Can we implement it using enum type?

Reply:
If the param is not defined with `required=True`, when the users don't input the param, the value of the this param would be `None`.